src/main/java/org/fundacionjala/sfdc/pageobjects/campaigns/SFCampNewModifyPage.java

```diff
@@ -0,0 +1,234 @@
+    }
+
+    /**
+     * @param campaignStatus setCampaignStatus.
+     */
+    private void setCampaignStatus(final String campaignStatus) {
+        CommonActions.selectOnComboBox(this.campaignStatus, campaignStatus);
+    }
+
+    /**
+     * @param campaignStartDate setCampaignStartDate.
+     */
+    private void setCampaignStartDate(final String campaignStartDate) {
+        CommonActions.setTextElement(this.campaignStartDate, campaignStartDate);
+    }
+
+    /**
+     * @param campaignExpectedRevenue setCampaignExpectedRevenue.
+     */
+    private void setCampaignExpectedRevenue(final String campaignExpectedRevenue) {
+        CommonActions.setTextElement(this.campaignExpectedRevenue, campaignExpectedRevenue);
+    }
+
+    /**
+     * @param campaignEndDate setCampaignEndDate.
+     */
+    private void setCampaignEndDate(final String campaignEndDate) {
+        CommonActions.setTextElement(this.campaignEndDate, campaignEndDate);
+    }
+
+    /**
+     * @param campaignBudgetedCost setCampaignBudgetedCost.
+     */
+    private void setCampaignBudgetedCost(final String campaignBudgetedCost) {
+        CommonActions.setTextElement(this.campaignBudgetedCost, campaignBudgetedCost);
+    }
+
+    /**
+     * @param campaignActualCost setCampaignActualCost.
+     */
+    private void setCampaignActualCost(final String campaignActualCost) {
+        CommonActions.setTextElement(this.campaignActualCost, campaignActualCost);
+    }
+
+    /**
+     * @param campaignExpectedResponse setCampaignExpectedResponse.
+     */
+    private void setCampaignExpectedResponse(final String campaignExpectedResponse) {
+        CommonActions.setTextElement(this.campaignExpectedResponse, campaignExpectedResponse);
+    }
+
+    /**
+     * @param campaignNumSent setCampaignNumSent.
+     */
+    private void setCampaignNumSent(final String campaignNumSent) {
+        CommonActions.setTextElement(this.campaignNumSent, campaignNumSent);
+    }
+
+    /**
+     * @param parentCampaign setParentCampaign.
+     */
+    private void setParentCampaign(final String parentCampaign) {
+        CommonActions.setTextElement(this.parentCampaign, parentCampaign);
+    }
+
+    /**
+     * @param campaignDescription setCampaignDescription.
+     */
+    private void setCampaignDescription(final String campaignDescription) {
+        CommonActions.setTextElement(this.campaignDescription, campaignDescription);
+    }
+
+    /**
+     * @param values contact field
+     * @return map of contact fields.
+     */
+    public Map<CampaignInputs, Value> getStrategyStepMap(final Map<CampaignInputs, String> values) {
+        EnumMap<CampaignInputs, Value> strategyMap = new EnumMap<>(CampaignInputs.class);
+
+        strategyMap.put(CampaignInputs.CAMPAIGN_NAME, () ->
+                this.setCampaignName(String.valueOf(values.get(CampaignInputs.CAMPAIGN_NAME))));
+        strategyMap.put(CampaignInputs.CAMPAIGN_ACTIVE, () ->
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this lambda with a method reference. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1612)